src/lightning/pytorch/core/hooks.py

```diff
@@ -412,55 +410,15 @@ def train_dataloader(self) -> TRAIN_DATALOADERS:
         - :meth:`setup`
 
         Note:
-            Lightning adds the correct sampler for distributed and arbitrary hardware.
+            Lightning tries to add the correct sampler for distributed and arbitrary hardware.
             There is no need to set it yourself.
-
-        Example::
-
-            # single dataloader
-            def train_dataloader(self):
```

Comment:
The LightningDataModule and data sections in the docs already use several examples with a full example such as this MNIST one

So I removed it from these docstring to make it more information-dense